### DISCUSSION TITLE
Problema Istanza ZigBee2MQTT

### DISCUSSION THREAD

**Discussion by omarantelmi** [indomus/forum#1434] [ZigBee, Z-Wave, Thread, Matter]
Salve a tutti
Posseggo un Raspberry pi4 e ho installato core su docker
Ho seguito passo passo la seguente guida: https://indomus.it/guide/come-installare-e-configurare-zigbee2mqtt-con-docker-su-raspbian-di-raspberry-pi/

Quando vado per eseguire il comando - docker container logs zigbee2mqtt -f - viene restituito tutto questo errore (o almeno credo, visto che non corrisponde con il log che indica la guida)
https://pastebin.com/z6LQQc5p

Potreste darmi una mano? Vi ringrazio anticipatamente

**Comment by indomus** [comment]
`ota: update_check_interval: 10`

controlla bene l'indentatura di questa porzione di configurazione

**Reply by omarantelmi** [reply]
Credo sia corretta, copiata e incollata dalla guida 
Ecco: https://pastebin.com/J9cCNUEq

**Reply by indomus** [reply]
è sbagliata, controlla bene

**Reply by snakuzzo** [reply]
aiutati con un linter (es. http://www.yamllint.com) ... troverai subito l'errore

**Reply by omarantelmi** [reply]
Si ti ringrazio, ho appena trovato l'errore...
Ora lancio il comando e mi viene restituito il seguente errore

Zigbee2MQTT:error 2022-02-02 16:50:00: MQTT failed to connect: Connection refused: Not authorized
Zigbee2MQTT:error 2022-02-02 16:50:00: Exiting...

Tutto il resto è verde...
Sto controllando la configurazione di Mosquitto e se eseguo il log docker ho questo errore:

1643817136: Client mqttjs_26f95d4b disconnected, not authorised.
1643817137: New connection from 172.18.0.1:53154 on port 1883.

**Reply by snakuzzo** [reply]
Stai andando avanti a copia-incolla?
Hai visto cosa ti dice? "Not Authorized" ... quindi probabilmente c'è un problema legato alla configurazione del broker MQTT e/o delle credenziali d'accesso inserite nella configurazione di zigbee2mqtt.
Ricontrolla.

**Reply by indomus** [reply]
raga però un po' di impegno va messo. ma se ti dice "non autorizzato", che potrà essere mai, su

**Reply by omarantelmi** [reply]
Scusate ma prima di postare e disturbare chiunque, cerco sempre una soluzione, leggendo e rileggendo. 
Avevo intuito che fosse un problema di autenticazione, ma ho ricontrollato i dati inseriti e corrispondono tutti.

Il dato che non mi torna è quell'indirizzo 172.18.0.1 ma credo che non abbia a che fare con me? E' un indirizzo di servizio esterno?

**Reply by indomus** [reply]
lascia stare l'ip, quello è relativo alla rete privata docker.
se ti risponde non autorizzato i casi sono due: o hai sbagliato username/password, oppure credi di usare credenziali corrette ma non lo sono. cmq il broker è up ed è raggiunto da zigbee2mqtt, altrimenti non ti risponderebbe proprio anziché darti errore di autenticazione.

concentrati sulla definizione dell'utenza di accesso al broker, una volta **certo** che le credenziali siano corrette, configura coerentemente zigbee2mqtt.

**Reply by KC-inDomus** [reply]
risolto?

**Comment by omarantelmi** [comment]
Ho risolto questa mattina
L'errore era contenuto qui: https://pastebin.com/J9cCNUEq
Gli asterischi prima di USER e PASS

**Reply by snakuzzo** [reply]
quelli non sono asterischi... e servono per commentare.
